toggles/webapp/toggles.ts

```diff
@@ -8,10 +8,9 @@ export default {
       defaultValue: true,
     },
     {
-      id: 'showCanvasLabels',
-      title: 'Show canvas labels on viewer thumbnails',
-      description:
-        'Shows the canvas label on the viewer thumbnails above the image number',
+      id: 'showHoldingsOnWork',
+      title: 'Shows holdings on the work page',
```

Comment:
Possibly my greatest nit comment ever, but I'd have 'Show' instead of 'Shows' in the title for this toggle ('shows' makes sense for the description). Feel free to ignore me, though.